generators/server/templates/build.gradle.ejs

```diff
@@ -384,7 +384,7 @@ dependencies {
         <%_ if (!reactive) { _%>
     implementation "org.springframework.boot:spring-boot-starter-data-jpa"
         <%_ } else { _%>
-    implementation "org.springframework.boot.experimental:spring-boot-starter-data-r2dbc:0.1.0.M3"
+    implementation "org.springframework.boot.experimental:spring-boot-starter-data-r2dbc:${spring_boot_starter_data_r2dbc_version}"
```

Comment:
This is not necessary, with Spring Boot 2.3 there's a new non-experimental starter, and the version will be managed by spring-boot-dependencies

Reply:
@murdos Yes I know. Wanted also to switch to r2dbc bom that I suggested in cbornet PR... let's remove it in SB 2.3

Reply:
Until we switch to SB 2.3, it is easier to use properties for updates.

Reply:
:( Since there won't be update (this experimental spring-boot-starter is not updated anymore), I don't see the point.
Now I need to rebase my spring-boot-2.3 branch and revert these changes in that branch.